stubs/tests/ResourceTest.php

```diff
@@ -51,12 +53,38 @@ public function test_users_can_edit_resources_belonging_to_them()
 
         $response = $this->actingAs($user)->put(localized_route('resources.update', $resource), [
             'title' => $resource->title,
-            'language' => $resource->language,
             'summary' => 'This is my updated resource.',
         ]);
         $response->assertRedirect(localized_route('resources.show', $resource));
     }
 
+    public function test_users_can_translate_resources_belonging_to_them()
+    {
+        if (! config('hearth.resources.enabled')) {
+            return $this->markTestSkipped('Resource support is not enabled.');
+        }
+
+        $user = User::factory()->create();
+        $resource = Resource::factory()->create(['user_id' => $user->id]);
+
+        $resource->setTranslation('title', 'en', 'title in English');
+        $resource->setTranslation('title', 'fr', 'title in French');
+
+        $this->assertEquals('title in English', $resource->title);
+        App::setLocale('fr');
```

Comment:
I think you will need to add this class via a `use` statement at the top of the test file. You can see what this looks like in the example code here: https://laravel.com/docs/9.x/localization#configuring-the-locale